registration/forms.py

```diff
@@ -105,7 +105,10 @@ class BetaTestApplicationForm(forms.ModelForm):
     """
     class Meta:
         model = BetaTestApplication
-        exclude = ('user', 'status', 'instance', 'accepted_privacy_policy')
```

Comment:
Is this related to the failing tests? All of the tests with user registration are passing the wrong default for `accept_privacy_policy` and `accept_terms`.

Reply:
Yes I think you're right. Perhaps in this case it would be better to keep using the exclude list?

Reply:
Hmm even when I add all the missing fields to the explicit fields list, it still fails those tests. I'm going to switch back to the exclude list method and disable pylint on it for now.

Reply:
Interestingly, those registration tests still fail with the exclude list set back to how it was originally. :thinking:

Reply:
@swalladge Sorry for sending you up a wrong tree. I'm not sure what the issue is. There are two commits: 6f60aa1ef23b0f34522db570dd628896ed58e885 and 727d8752ae48b60350b260c9b2e8f430ce24b034 , which add a default of "checked" for those two boxes during registration. You have both on this branch.

Reply:
@SSPJ I believe I discovered the issue - it was related to the form changes and how the value was being parsed. I made a [note of the other related issues](https://github.com/open-craft/opencraft/pull/436#issuecomment-510716910), but didn't attempt to fix them because this is because of the temporary removal of django angular forms. This issue I was able to fix because it was a general incorrectness in how the value was parsed.

Reply:
Good work sleuthing that out, @swalladge !